cmd/soroban-rpc/internal/config/version.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+var (
+	// Version is the soroban-rpc version number, which is injected during build time.
+	Version = "0.1.0"
```

Comment:
Suggest defaulting this to something that appears more invalid.
```suggestion
	Version = "0.0.0"
```